src/Cli.ts

```diff
@@ -0,0 +1,9 @@
+import { ArgumentParser, Namespace } from 'argparse';
+
+export const parseArgs = (): Namespace => {
+	const parser = new ArgumentParser();
+
+	parser.add_argument('-v', '--version', { action: 'store_true' });
+
+	return parser.parse_args() as Namespace;
+};
```

Comment:
maybe rename file `cli` or `parseArgs`? `Cli` makes me think its a class with that name

Reply:
Yea i totally agree. I'll change it to lowercase.